doc/source/examples/diffractionobjectsexample.rst

```diff
@@ -0,0 +1,41 @@
+.. _Diffraction Objects Example:
+
+:tocdepth: -1
+
+Diffraction Objects Example
+###########################
+
+This example will demonstrate how to use the ``DiffractionObject`` class in the
+``diffpy.utils.scattering_objects.diffraction_objects`` module to process and analyze diffraction data.
+
+1) We have the function ``q_to_tth`` to convert q to two theta values in degrees, and ``tth_to_q`` to do the reverse.
+   You can use these functions with a pre-defined ``DiffractionObject``. ::
+
+    # convert q to tth
+    from diffpy.utils.scattering_objects.diffraction_objects import DiffractionObject
+    test = DiffractionObject(wavelength=1.54)
```

Comment:
instead of test let's call it `my_diffraction_pattern`

We are not writing all of the docs here, so you can start with something  like "assuming we have created a `DiffractionObject` called `my_diffraction_pattern` from a measured diffraction pattern, and we have specified the wavelenth (see Section ??)  we can use the `q_to_tth` and `tth_to_q` functions to convert between Q and two-theta by typing `my_diffraction_pattern.q_to_tth()`..." ..and so on.   Let's make an issue to add to docs "How to set a wavelength" and how to instantiate a diffraction object and how to add a diffraction pattern.